string source = rogr
string target = camera2
time timestamp
